package.json

```
@@ -44,12 +44,12 @@
   "devDependencies": {
     "@types/react": "^16.9.0",
     "husky": "^0.14.3",
-    "prettier": "1.14.2",
+    "prettier": "^1.19.0",
     "pretty-quick": "^1.6.0",
-    "react": "~16.9.0",
-    "react-dom": "~16.9.0",
-    "rimraf": "~2.6.1",
-    "typescript": "~3.7.0"
+    "react": "^16.13.1",
```

These could be bumped to:

```suggestion
    "react": "^17.0.0",
```

With the latest lab rc.

---

Thanks for the suggestion

Actually I went for `^16.13.1 || ^17.0.0` as there is no incompatibility between the two versions for this package.